Tools/AP_Bootloader/board_types.txt

```diff
@@ -266,6 +266,7 @@ AP_HW_MicoAir405v2                   1150
 AP_HW_ORAQF405PRO                    1155
 AP_HW_CBU_StampH743                  1156
 AP_HW_FOXEERF405_V2                  1157
+AP_HW_NxtPX4v2                       1159
```

Comment:
I suggest creating a PR to reserve an ID in case this PR takes a while to get in.

Reply:
Thank you for your suggestion. I have cerated the PR: [#26746](https://github.com/ArduPilot/ardupilot/pull/26746)